website/snippets/_fusion-troubleshooting.md

```diff
@@ -1,5 +1,48 @@
 If you run into any issues, check out the troubleshooting section below.
 
+<Expandable alt_header="How to create a .dbt directory and move your file">
+
+If you've never had a `.dbt` directory, you should perform the following recommended steps to create one. If you already have a `.dbt` directory, move the `dbt_cloud.yml` file into it.
+
+<Tabs>
+<TabItem value="Create a .dbt directory">
+
+  1. Clone your dbt project repository locally.
+  2. Use the `mkdir` command followed by the name of the folder you want to create. Add the `~` prefix to create a `.dbt` folder in the root of your filesystem:
+
+     ```bash
+     mkdir ~/.dbt
+     ```
+
+This will create a `.dbt` folder in the root directory.
+
+For Mac users, since it's a hidden folder (due to the dot prefix), it won't be visible in Finder by default. To view hidden files and folders, press Command + Shift + G.
```

Comment:
This command opens the "Go to Folder" search bar, it's not a shortcut to display hidden folders in the Finder window